### ISSUE TITLE
Inaccessible HTML used for form labels

### ISSUE THREAD

**Issue by m9n** [Crocoblock/suggestions#1538]
**Describe the bug**

Form labels are not marked up with `<label>`.  They are marked up with `<div>` tags, which carry no semantic meaning, harm form accessibility, and make it harder or impossible to style some elements.

**To Reproduce**

1. Create a form with JetForms.  Ensure it has at least one field with a label.
2. Add this form to a page with Elementor.
3. Save and view the page from the front-end.
4. Try clicking on a label - your cursor will not be placed in the field, as per expected behaviour with HTML forms.
4. Go into your browser's inspector panel and inspect the HTML for the label.  You'll find it's built only with asemantic DIVs and SPANs.

**Expected behavior**

- Form labels should be built with the <label> tag.
- When you click or touch a label, your browser should focus/trigger that form element.
- With <label> tags, we'd have far more styling options for form elements, particularly for elements which are difficult to style such as file inputs.
- I expect to be able to make websites which [satisfy WCAG](https://www.w3.org/TR/WCAG20-TECHS/H44.html).  The UK's Disability Discrimination Act and the USA's ADA, 504 and 508 laws require passing WCAG 2.0.

I'd suggest the following HTML to be produced, which with minimal CSS I'd expect to be easy to implement and backwards compatible with existing websites:

```
<div class="jet-form__label">
	<label class="jet-form__label-text" for="field_id">Image</label>
</div>
```

I noticed that your solution doesn't currently include an ID attribute on form elements.  This must also be added to support correct labelling.

It would also be useful to have the field ID present as an extra CSS class in the field wrapper.  This way we'd be able to target specific elements with CSS, whereas right now we have to use unstable techniques such as `nth-child` to do this.

**Platforms / devices:**

Affects all platforms and devices because this is a markup issue.

**Additional context**

https://www.w3.org/WAI/tutorials/forms/labels/
https://cielo24.com/2017/03/a-brief-history-of-accessibility-law-us/
https://webaim.org/techniques/forms/
